AnnoyancesFilter/sections/popups.txt

```diff
@@ -236,6 +236,7 @@ technopark.ru#$#.tp-modal-backdrop { display: none !important; }
 technopark.ru#$#body { overflow: auto !important; }
 uniqlo.com##.subscribe-popup-overlay
 uniqlo.com##.subscribe-popup
+oneforall.de###sliding-popup
```

Comment:
It is cookies related. Added to existing `ladige.it,rovio.com,aida64.com,sat1spiele.de##body > #sliding-popup`